packages/location-management/src/components/LocationForm/utils.tsx

```diff
@@ -135,18 +135,16 @@ export const getLocationFormFields = (
 
 /** removes empty undefined and null objects before they payload is sent to server
  *
- * @param {any} obj object to remove empty keys from
+ * @param {Dictionary} obj object to remove empty keys from
  */
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function removeEmptykeys(obj: any) {
+export function removeEmptykeys(obj: Dictionary) {
   Object.entries(obj).forEach(([key, value]) => {
     if (typeof value === 'undefined') delete obj[key];
     else if (value === '' || value === null) delete obj[key];
+    // if datatype is object this clearly means that either the value is an array or a json object
     else if (typeof value === 'object') {
-      // if datatype is object this clearly means that either the value is an array or a json object
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      const valueObj = value as { [key: string]: any } | any[];
-      if (typeof valueObj.length !== 'undefined' && valueObj.length === 0) delete obj[key];
+      // If its any array with length 0 (only array have length property)
+      if (typeof value.length !== 'undefined' && value.length === 0) delete obj[key];
```

Comment:
would `value.length === 0` only suffice here?

Reply:
Just updated the checks for better type checking